resources/views/v2/components/FrontpageSearchRow/FrontpageSearchRow.css

```diff
@@ -0,0 +1,66 @@
+@import "variables";
```

Comment:
lot of $non-variables and hardcoded px values. Can be followed up later as cleanup PR